src/lib.rs

```diff
@@ -346,10 +345,425 @@ impl From<Error> for io::Error {
     }
 }
 
+/// Methods that return information about a path.
+///
+/// This trait provides the familiar methods from `std::path::Path`
+/// for the `Path*` types. These methods take the same parameters and return
+/// the same types as the originals in the standard library, except where
+/// noted.
+///
+/// As a general rule, methods that can return an error will return a rich
+/// [`path_abs::Error`] instead of a [`std::io::Error`].
```

Comment:
worth noting that the error will be autoconverted with `?` or `into()`.

Reply:
Should be fixed in 671e918